lib/fact_check_email_handler.rb

```diff
@@ -39,13 +39,16 @@ def process_message(message)
 
   # takes an optional block to call after processing each message
   def process
+    email_count = 0
     Mail.all(read_only: false, delete_after_find: true) do |message|
-      message.skip_deletion unless process_message(message)
+      message.mark_for_delete = process_message(message)
```

Comment:
I wonder if these lines could be combined into a single conditional. Perhaps something like:

```ruby

if message.mark_for_delete = process_message(message)
  email_count += 1
end
```

Reply:
That would work too, but I don't think it would add anything in terms of clarity or brevity.